ClientSideEncryptionToCPKN/ClientSideKeyVaultKeyToMicrosoftManagedKey/samplesrc/samplesetup/Setup.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage;
+using System;
+using System.IO;
+using System.Diagnostics;
+
+namespace keyVaultClientSideToMicrosoftManagedServerSide
+{
+    class Setup
+    {
+        //Creates example container, client side encrypted blob, and encryption scope for sample
+        public static void SetupForExample(
+            BlobServiceClient blobService,
+            string containerName,
+            string fileName,
+            string encryptionScopeName,
+            ClientSideEncryptionOptions clientSideOption)
+        {
+            //Create example Container and .txt file, upload .txt file as client side encrypted blob
+            BlobContainerClient containerClient = blobService.CreateBlobContainer(containerName);
+            //Create BlobClient with Client Side Encryption Options to upload client side encrypted data
+            BlobClient blobClient = containerClient.GetBlobClient(fileName).WithClientSideEncryptionOptions(clientSideOption);
+
+            //Upload blob with client side encryption
+            using FileStream uploadFileStream = File.OpenRead(Path.Combine(Constants.samplePath, Constants.fileName));
+            blobClient.Upload(uploadFileStream, true);
+            uploadFileStream.Close();
+            Console.WriteLine("Uploaded to Blob storage as blob: \n\t {0}\n", blobClient.Uri);
+
+            CreateEncryptionScopeMMK(encryptionScopeName);
+        }
+
+        //Delete files in the Data folder            
+        public static void CleanUp(string path)
+        {
+            Directory.Delete(path, true);
+        }
+
+        //Runs Azure CLI script that creates an encryption scope that uses Microsoft managed keys
+        private static void CreateEncryptionScopeMMK(string encryptionScopeName)
+        {
+            Process process = Process.Start("CMD.exe", "/C az storage account encryption-scope create --name " + encryptionScopeName
+                + " -s Microsoft.Storage --account-name " + Constants.storageAccount + " -g " + Constants.resourceGroup + " --subscription " + Constants.subscriptionId);
+            process.WaitForExit();
```

Comment:
Instead of doing that. why don't we just create setup.cmd which will do this and call this EXE ?